src/duct.py

```diff
@@ -157,17 +168,26 @@ def finalize(self):
             print(Colors.OKGREEN)
         else:
             print(Colors.FAIL)
-
-        print("-----------------------------------------------------")
-        print("                    duct report")
-        print("-----------------------------------------------------")
         print(f"Exit Code: {self.process.returncode}")
         print(f"{Colors.OKCYAN}Command: {self.command}")
+        print(f"Log files location: {self.output_prefix}")
         print(f"Wall Clock Time: {self.elapsed_time}")
-        print(f"Number of Processes: {len(self.max_values)}")
-        for pid, values in self.max_values.items():
-            values.pop("timestamp")  # Meaningless
-            print(f"    {pid} Max Usage: {values}")
+        print(
+            f"Memory Peak Usage: {self.max_values.get('totals', {}).get('pmem', 'unknown')}%"
+        )
+        print(
+            f"CPU Peak Usage: {self.max_values.get('totals', {}).get('pcpu', 'unknown')}%"
+        )
```

Comment:
in addition to 
- #36 
- 

which is about "very fast" programs, here we seems get nothing  even not for so short programs:

```shell
❯ duct -- ./test_script.py --duration 3
duct is executing ./test_script.py --duration 3...
Log files will be written to .duct/logs/2024.06.04T16.48.06-850972_
this is of test of STDERR: ERRRRRRRRRRRRRRR
Test completed. Consumed 10 MB for 3 seconds with CPU load factor 10000.

Exit Code: 0
Command: ./test_script.py --duration 3
Log files location: .duct/logs/2024.06.04T16.48.06-850972_
Wall Clock Time: 5.056247234344482
Memory Peak Usage: unknown%
CPU Peak Usage: unknown%
```

so we ran for 3 seconds but there is unknown CPU and memory usage

Reply:
This occurred when runtime < report interval. Fixed by https://github.com/con/duct/pull/37/commits/26bf7d4891300b68ee1f0f35343bd5088d99df55